tutorials/toolkits/mplot3d.py

```diff
@@ -24,10 +24,6 @@
    :target: ../../gallery/mplot3d/subplot3d.html
    :align: center
 
-.. versionchanged:: 1.0.0
-   Prior to Matplotlib 1.0.0, only a single `.Axes3D` could be created per
-   figure; it needed to be directly instantiated as ``ax = Axes3D(fig)``.
```

Comment:
This is so ancient that there is little value in keeping, and the stated way of creation is not recommended anymore. So it's simplest to remove completely.